jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -814,88 +814,77 @@ def _fft_rng_factory(dtype):
   for window_dilation in [(1, 1, 1, 1)]
 )
 
-lax_reduce_window = tuple(
-  # Tests with 2d shapes (see tests.lax_test.testReduceWindow)
-  Harness(f"2d_shape={jtu.format_shape_dtype_string(shape, dtype)}_initvalue={init_value}_computation={computation.__name__}_windowdimensions={window_dimensions}_windowstrides={window_strides}_padding={padding}_basedilation={base_dilation}_windowdilation={window_dilation}".replace(' ', ''),
-          lax.reduce_window,
-          [RandArg(shape, dtype), StaticArg(init_value), StaticArg(computation),
-           StaticArg(window_dimensions), StaticArg(window_strides),
-           StaticArg(padding), StaticArg(base_dilation), StaticArg(window_dilation)],
-          shape=shape,
-          dtype=dtype,
-          init_value=init_value,
-          computation=computation,
-          window_dimensions=window_dimensions,
-          window_strides=window_strides,
-          padding=padding,
-          base_dilation=base_dilation,
-          window_dilation=window_dilation)
-  for computation in [lax.add, lax.max, lax.min, lax.mul]
-  for dtype in { lax.add: filter(lambda t: t != np.bool_, jtu.dtypes.all)
-               , lax.mul: filter(lambda t: t != np.bool_, jtu.dtypes.all)
-               , lax.max: jtu.dtypes.all
-               , lax.min: jtu.dtypes.all
-               }[computation]
-  for init_value in map(
-      dtype,
-      (lambda ts: ts[0] if not dtype in jtu.dtypes.all_floating else ts[1])(
-          { lax.add: ([0, 1], [0, 1])
-          , lax.mul: ([1], [1])
-          , lax.max: ([1], [-np.inf, 1])
-          , lax.min: ([0], [np.inf, 0])
-          }[computation]
-      )
-  )
-  for shape in [(4, 6)]
-  for window_dimensions in [(1, 2)]
-  for window_strides in [(2, 1)]
-  for padding in tuple(set([tuple(lax.padtype_to_pads(shape, window_dimensions,
-                                                      window_strides, p))
-                            for p in ['VALID', 'SAME']] +
-                           [((0, 3), (1, 2))]))
-  for base_dilation in [(2, 3)]
+def _make_reduce_window_harness(name, *, shape=(4, 6), base_dilation=(1, 1),
+                                computation=lax.add, window_dimensions=(2, 2),
+                                window_dilation=(1, 1), init_value=0,
+                                window_strides=(1, 1), dtype=np.float32,
+                                padding=((0, 0), (0, 0))):
+  return Harness(f"{name}_shape={jtu.format_shape_dtype_string(shape, dtype)}_initvalue={init_value}_computation={computation.__name__}_windowdimensions={window_dimensions}_windowstrides={window_strides}_padding={padding}_basedilation={base_dilation}_windowdilation={window_dilation}".replace(' ', ''),
+                 lax.reduce_window,
+                 [RandArg(shape, dtype),
+                  StaticArg(np.array(init_value, dtype=dtype)),
```

Comment:
Is it important for the init_value to be a compile-time constant? If not, I think it is better to have it a dynamic argument (just drop `StaticArg`).